src/main/java/Todos.java

```diff
@@ -0,0 +1,28 @@
+public class Todos {
+
+    public String item;
+    public boolean marked;
+
+    public String type;
+
+    Todos(String name, boolean marked, String type) {
+        this.item = name;
+        this.marked = marked;
+        this.type = type;
+    }
+
+    public String getType() {
+        return "[T]";
+    }
+    public void setMark() {
+        this.marked = true;
+    }
+
+    public void unMark() {
+        this.marked = false;
+    }
+
+    public boolean is_marked() {
```

Comment:
For methods, you should be using camelCase, ie. isMarked(), similar to the rest of the methods in this file